.readthedocs.yml

```diff
@@ -13,7 +13,6 @@ python:
       extra_requirements:
         - docs
         - all
-    - requirements: docs/rtd_requirements.txt
```

Comment:
We wouldn't know about this one until after merge, unless you want to try build RTD from your fork...